openwisp_radius/api/freeradius_views.py

```diff
@@ -525,6 +533,39 @@ def send_radius_accounting_signal(self, accounting_data):
             view=self,
         )
 
+    def _handle_accounting_on(self, data):
+        """
+        Handle Accounting-On packets by triggering a background task
+        to close stale sessions for the NAS.
+
+        When a NAS (Network Access Server) sends an Accounting-On packet,
+        it indicates that the NAS has restarted or experienced a power failure.
+        All active sessions on that NAS should be considered stale and closed.
+        """
+        nas_ip_address = data.get("nas_ip_address")
+        if not nas_ip_address:
+            logger.warning("Accounting-On packet received without nas_ip_address")
+            return
+
+        organization_id = self.request.auth
+        if not organization_id:
+            logger.warning("Accounting-On packet received without valid organization")
+            return
+
+        # Import here to avoid circular imports
+        from ..tasks import handle_accounting_on
+
+        # Execute the task in the background to avoid blocking the API response
+        # and to prevent performance issues when there are many sessions to close
+        handle_accounting_on.delay(
+            organization_id=organization_id, nas_ip_address=nas_ip_address
+        )
+
+        logger.info(
```

Comment:
Let's use `logger.debug`, this would be the normal behavior, we shouldn't emit output for standard behavior.